### PR TITLE
modules test and README doc

### PR DESCRIPTION
**Pull request by geewynn**
Made some changes to the k8s modules
Added a quick setup guide in the example folder

### CODE REVIEW

File: aks_cluster.tf [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+  location = var.resource_group_location
+}
+
 resource "azurerm_kubernetes_cluster" "k8s" {
   name                = "${var.cluster_name}-cluster"
-  location            = var.resource_group_name
-  resource_group_name = var.resource_group_location
+  location            = var.resource_group_location
+  resource_group_name = var.resource_group_name
   dns_prefix          = "k8scluster"
```
Comment (charlesa101) [inline aks_cluster.tf]: can this be a variable

Comment (geewynn) [reply aks_cluster.tf]: it is a variable

File: aks_cluster.tf [hunk 2]
```diff
@@ -29,17 +35,16 @@ resource "azurerm_kubernetes_cluster" "k8s" {
     client_secret = var.client_secret
   }
 
+  role_based_access_control {
+    enabled = true
+  }
+
   network_profile {
     network_plugin    = "kubenet"
     load_balancer_sku = "Standard"
```
Comment (charlesa101) [inline aks_cluster.tf]: this should be parameterized

Comment (geewynn) [reply aks_cluster.tf]: Done

File: aks_cluster.tf [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
   default_node_pool {
-    name                = var.node_pool_name
-    node_count          = var.node_pool_count
-    vm_size             = var.node_pool_vm_size
-    type                = "VirtualMachineScaleSets"
-    os_disk_size_gb     = var.node_pool_osdisk_size
-    enable_auto_scaling = true
-    max_count           = var.node_pool_max_count
-    min_count           = var.node_pool_min_count
+    availability_zones   = [1]
```
Comment (charlesa101) [inline aks_cluster.tf:13]: whats this variable?

Comment (geewynn) [reply aks_cluster.tf:13]: @KennedyUC

File: example/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+## Quick Guide to running the terraform module
+
```
Comment (charlesa101) [inline example/README.md:58]: steps to create the az credentials etc are not here? the steps went through yesterday

Comment (geewynn) [reply example/README.md:58]: missed that, adding now, give me a minute

Comment (geewynn) [reply example/README.md:58]: added

File: aks_cluster.tf (hunk 3 above)
Comment (kennedyuche) [reply aks_cluster.tf:13]: Allows for AKS nodes to run across Availability Zones. Forms a protection against any single failure of a datacenter. Just used 1 for testing the k8s cluster.

**Pull request merged**